Jenkinsfile

```diff
@@ -178,8 +178,10 @@ pipeline {
                        directory: 'scons_local',
                        scm: [url: 'https://github.com/libfuse/libfuse.git',
                              branch: "${env.FUSE_COMMIT}",
+                             checkoutDir: '${WORKSPACE}/fuse',
```

Comment:
The default is to checkout to _build.external/[target].   The problem is that this gets removed by the scm checkout code in a "clean" step.   This change should probably be made to other builds in this file but I wanted to solve the immediate problem here which was that the fuse build was blocking my change.

Reply:
Filed CORCI-665 for this issue